cloud/home/index.md

```diff
@@ -0,0 +1,36 @@
+---
+title: Home | Terramate Cloud
+description: Learn about the Home page of the Terramate Cloud dashboard
+---
+
+# Home Page
+
+The Home page provides a summary of your engagements in Terramate Cloud, including:
+
+- **Alerts**: Open alerts assigned to you.
+- **Pull Requests**: Open pull requests authored by you.
+- **Deployments**: Deployments in which you collaborated.
+
+![Home Page](../assets/home_tmc.png "Home Page")
+
+
+## Alerts
+
+The Alerts list displays details of alerts assigned to you, following the same layout as the [Alerts dashboard](../alerts/index.md).
+
+
+## Pull Requests
+
+The Pull Requests list shows [pull requests](../previews/index.md) you authored in repositories associated with your Terramate Cloud organization.
+
+
+## Deployments
+
+The Deployments list presents deployments where you contributed by pushing commits. The list shows:
```

Comment:
The relation on contributions is different. 
```suggestion
The Deployments list presents deployments where you contributed by creating or reviewing the related pull request or triggering the deployment. The list shows:
```
Please rephrase it the best way you think of.

Reply:
thanks @sixstone-qq for the more correct description.